### DISCUSSION TITLE
Discussion

### DISCUSSION THREAD

**Discussion by LoriaLawrenceZ** [ZimbresAguilar/RPG-Translator-Observador#1] [General]
@Paiva8859 , qnd a gente estiver no senai ou fazendo algum projeto pela nossa """""empresa""""" br fazer por aqui, p ficar um bgl mais profissionalzinho

**Comment by LoriaLawrenceZ** [comment]
Aí mn, br só deletar os arquivos q eu tinha criado e começar do 0 o site de tradutor

**Reply by Paiva8859** [reply]
beleza

**Comment by Paiva8859** [comment]
Vamos fazer a aplicação web ent?

**Reply by Paiva8859** [reply]
@LoriaLawrenceZ

**Comment by ArselinaLuqman** [comment]
ok

**Comment by LoriaLawrenceZ** [comment]
Where you all guys coming from

**Reply by LoriaLawrenceZ** [reply]
All of sudden you guys started to appear simply out of nowhere, like, it was supposed to only me and my partner to have access to this repository for some time
